Code/src/main/java/nl/utwente/viskell/model/Wire.java

```diff
@@ -0,0 +1,14 @@
+package nl.utwente.viskell.model;
+
+public class Wire {
```

Comment:
This seems at odds with use of the term 'Wire' in #251 - this would be represented by connections, not wires, in the UI, right?